config/valid_email.yml

```diff
@@ -9,3 +9,7 @@ disposable_email_services:
   - usimmigrationemail.com
   - firemailbox.club
   - safemail.icu
+  - temp2-login.com
+  - temp5-login.com
+  - temp9-login.com
+  - temp13-login.com
```

Comment:
Also I think it's time to move this file to be a key in `application.yml` so we can roll it out more quickly and with less noise

Reply:
https://github.com/18F/identity-idp/pull/5754